scripts/hauler_manifest.yaml

```diff
@@ -0,0 +1,70 @@
+# This is a base file to be used to import all the required resources to run
+# Kubewarden in airgap environment into Hauler.
+#
+# To sync this file with Hauler and load everything into its storage, you can
+# run:
+# hauler store sync --filename scripts/hauler_manifest.yaml
+apiVersion: content.hauler.cattle.io/v1
+kind: Images
+metadata:
+  name: kubewarden-container-images
+  annotations:
+    hauler.dev/certificate-oidc-issuer: https://token.actions.githubusercontent.com
+spec:
+  images:
+    - name: ghcr.io/kubewarden/audit-scanner:v1.27.0
+      certificate-identity-regexp: https://github.com/kubewarden/audit-scanner/.github/workflows/release.yml@refs/tags/v1.27.0
+    - name: ghcr.io/kubewarden/kubewarden-controller:v1.27.0
+      certificate-identity-regexp: https://github.com/kubewarden/kubewarden-controller/.github/workflows/release.yml@refs/tags/v1.27.0
+    - name: ghcr.io/kubewarden/policy-server:v1.27.0
+      certificate-identity-regexp: https://github.com/kubewarden/policy-server/.github/workflows/release.yml@refs/tags/v1.27.0
+---
+# The policies are in a separated definition just to allow a better keyless validation
+# without the need to duplicate configuration
+apiVersion: content.hauler.cattle.io/v1
+kind: Images
+metadata:
+  name: kubewarden-policies
+  annotations:
+    hauler.dev/certificate-oidc-issuer: https://token.actions.githubusercontent.com
+    hauler.dev/certificate-identity-regexp: https://github.com/kubewarden/github-actions/.github/workflows/.*
+spec:
+  images:
+    - name: ghcr.io/kubewarden/policies/allow-privilege-escalation-psp:v1.0.0
+    - name: ghcr.io/kubewarden/policies/capabilities-psp:v1.0.3
+    - name: ghcr.io/kubewarden/policies/host-namespaces-psp:v1.0.3
+    - name: ghcr.io/kubewarden/policies/hostpaths-psp:v1.0.3
+    - name: ghcr.io/kubewarden/policies/pod-privileged:v1.0.3
+    - name: ghcr.io/kubewarden/policies/user-group-psp:v1.0.2
+---
+# The policy reporter and kuberlr images are defined in the dedicated manifest section because
+# the container images are not signed. Therefore, this difference is very clear
```

Comment:
policy-reporter images and charts are definitely signed via keyless cosign (see [here](https://github.com/kyverno/policy-reporter/blob/main/.github/workflows/release-image.yaml) and [here](https://github.com/kyverno/policy-reporter/blob/main/.github/workflows/release-chart.yaml)), and kuberlr-kubectl should be too (IIRC it happens when calling `ecm-distro-tools/publish-image`).

Reply:
kuberlr-kubectl is signed only when pushed to the [prime registry](https://github.com/rancher/ecm-distro-tools/blob/3fb58c0204d3fc185516c175b6c58adc3089bb86/actions/publish-image/action.yaml#L188-L206). Which is not the case [here](https://github.com/rancher/kuberlr-kubectl/blob/b030093dd5d6457ac4d262d4fb8c9f7756f25145/.github/workflows/release.yml#L74C12-L74C31)